backend/sirius-web-graphql-utils/pom.xml

```diff
@@ -47,7 +48,7 @@
 		<dependency>
         	<groupId>com.graphql-java</groupId>
         	<artifactId>graphql-java</artifactId>
-        	<version>14.0</version>
+        	<version>${graphql-java.version}</version>
```

Comment:
Could you fix the problem in the appropriate commit please? Everything is fine apart from that.

Reply:
I've squashed all the GraphQL updated into a single commit (14.0 => 16.1).